unstructured/ingest/cli/cmds/astra.py

```diff
@@ -38,10 +38,25 @@ def get_cli_options() -> t.List[click.Option]:
             ),
             click.Option(
                 ["--embedding-dimension"],
+                required=True,
                 default=384,
                 type=int,
                 help="The dimensionality of the embeddings",
             ),
+            click.Option(
+                ["--namespace"],
+                required=False,
+                default=None,
+                type=str,
+                help="The Astra DB namespace to write into.",
+            ),
+            click.Option(
+                ["--requested-indexing-policy"],
+                required=False,
+                default=None,
+                type=str,
```

Comment:
import Dict at the top 
`from unstructured.ingest.cli.interfaces import CliConfig, Dict`

```
type=Dict(),
help="The indexing policy to use for the collection."
'example: \'{"blablabla":"blablabla"}\' ',
```

Reply:
Updated!